test/overrides/nested/dontlist.txt

```diff
@@ -0,0 +1 @@
+don't!
```

Comment:
What's the point of this?  Some sort of check to make sure it doesn't recurse into sub directories?